### ISSUE TITLE
User-friendly 'about' link

### ISSUE THREAD

**Issue by mialondon** [UniversalViewer/universalviewer#1309]
## So that I can learn about the UV project

Currently 'more info' goes to a github page, which is ok but not brilliant.

**Comment by Geoffsc** [comment]
As we discussed on the call, one option would be to change the more info link to https://universalviewer.io. Although as we also discussed this website needs work. Would that be a "good enough" fix for now and when the website is updated it will further improve things? That would be a relatively easy/quick fix.

**Comment by mialondon** [comment]
Yes, the minimum fix was to point to the .io site for now, and separately work to update the website.

If there's anyone with the capacity to improve the content of https://github.com/UniversalViewer/universalviewer/wiki/About (linked to from the .io site) in this sprint, we could link there instead, which reduces the dependency on the .io site.

**Comment by mialondon** [comment]
@demiankatz I like that!

**Comment by mialondon** [comment]
I've rewritten some of the 'about' page (I'm on a long 'listening' call) but it doesn't really sell the software at the moment!

**Comment by erinburnand** [comment]
Agree @demiankatz, i really like that too! Assume we will do similar for #1310

**Comment by scottjenson** [comment]
Whichever link we use, can be log its use? It would be helpful to know how many times people actually click on that link? Maybe a simple redirect would be easy to do?

My point is that it would be very helpful to know, after X months, how many times that link is actually clicked on. My fear is that it will be shockingly low.

**Comment by demiankatz** [comment]
@scottjenson, I agree that this think seems very unlikely to be used very much, given its hidden location and the fact that most people won't care. I don't think we currently have any web analytics infrastructure for the project; that might be something for the steering group to take up. As you say, it's easy in theory to measure clicks with a redirect, a unique hash, or some similar mechanism -- but I don't think we have the infrastructure in place yet, and I don't know if it's worth building solely to confirm what I think is nearly a foregone conclusion. :-)

**Comment by scottjenson** [comment]
Agree, we don't want to add too much to this sprint. But as you say, in principle, having the ability to track link usage is a good thing. I would only put this on the roadmap if we agree to find a bare bones/cheap way to do this. I wouldn't want this to turn into a design-the-best-possible-link-tracker type of subcommittee.

**Comment by mialondon** [comment]
The BL team and I assume others have integrated Google Analytics into their local instances. @erinburnand perhaps the team could document that process?

Back in the day, Wellcome used their analytics really well to understand which features were most used, but I'm not sure anyone else has ever had the capacity to study their analytics in detail.

**Comment by demiankatz** [comment]
@scottjenson, I don't think the analytics piece needs to hold up the generalization of the about link. If we create a configurable about link, we can reconfigure its default to use an analytics-friendly link later, even if we don't have the capacity to start with one.

**Comment by LlGC-szw** [comment]
Localised URL and labels needed.

**Comment by demiankatz** [comment]
> The BL team and I assume others have integrated Google Analytics into their local instances. [@erinburnand](https://github.com/erinburnand) perhaps the team could document that process?
> 
> Back in the day, Wellcome used their analytics really well to understand which features were most used, but I'm not sure anyone else has ever had the capacity to study their analytics in detail.

The UV's event system was designed in part to make it easier to capture this type of data. I'm not sure how enthusiastic we want to be about Google Analytics from a data privacy/ethics perspective... but there are also alternatives like Matomo that can do the same thing in a different way. It takes time to set up the infrastructure, and even more time to analyze the data, though, so it's just a matter of deciding what we want to commit to. While I fully recognize the value of analytics, it's less interesting to me personally than more direct problem-solving, so I'm not going to be the first person to volunteer to help with it when there are so many other things I could be potentially doing. But if you get the right volunteer with the right enthusiasm, there's certainly potential to do useful things.

**Comment by mialondon** [comment]
There's also Umami, etc, but in many cases folk might be stuck with the institutional solution (as we are).

**Comment by demiankatz** [comment]
> There's also Umami, etc, but in many cases folk might be stuck with the institutional solution (as we are).

I think there are two separate questions here:

1.) How can local institutions measure the usage of their UV instances?

2.) How can the UV project itself measure usage of certain features across all instances?

I think, generally speaking, people don't like it when projects intrusively measure things. But in the case of the about link, measuring how often people come from the settings dialog to the UV website is probably not a terrible invasion of privacy.

**Comment by demiankatz** [comment]
@Geoffsc pointed out at today's post-standup meeting that the present implementation of the outgoing UV website link is actually a full HTML `<a>` tag, so both the URL and the label are already configurable through the locale settings. If we're okay with that implementation, there's actually not really any additional technical work needed, since we already have the ability to fully customize that link on a locale-specific basis. I admit that this solution feels a little hacky to me -- I'd prefer to separate internationalization from specific HTML markup -- but as a practical matter, since the precedent is already set, it's a lot easier to take advantage of it than to build a whole new system with separate translation strings and URL settings. What does everyone think? Are revisions needed, or should we run with it as-is?

In any case, @Geoffsc noticed that for some reason, the URL had an anchor on it in some locales but not others; this seemed unintentional and has been standardized by #1319.
